src/mapper/bearing_detail_mapper.py

```diff
@@ -1,13 +1,16 @@
 from mapper.sensor_summary_mapper import to_sensor_list
 from mapper.machine_summary_mapper import to_machine_summary
-
+from repositories.mongo_config import HEADER
 
 def to_bearing_detail(bearing):
-    return {
-        "id": str(bearing.id),
-        "status": bearing.alert_status,
-        "tags": bearing.bearing_tags,
-        "sensorsId": bearing.sensors_id_list,
-        "sensors": to_sensor_list(bearing.sensors_id_list),
-        "machineInfo": to_machine_summary(bearing.machine)
-    }
+    try:
+        return {
+            "id": str(bearing.id),
+            "status": bearing.alert_status,
+            "tags": bearing.bearing_tags,
+            "sensorsId": bearing.sensors_id_list,
+            "sensors": to_sensor_list(bearing.sensors_id_list),
+            "machineInfo": to_machine_summary(bearing.machine)
+        }
+    except:
```

Comment:
`except` range too broad, the best practice would be to only catch an exception that you can handle. In this case the exception you would wanna catch is `AttributeError`